Orange/canvas/__main__.py

```diff
@@ -395,12 +348,30 @@ def handle_response(result):
         question.finished.connect(handle_response)
         question.show()
 
-    with redirect_stdout(stdout), redirect_stderr(stderr):
-        log.info("Entering main event loop.")
-        try:
+    # Tee stdout and stderr into Output dock
+    output_view = canvas_window.output_view()
+
+    stdout = TextStream()
+    stdout.stream.connect(output_view.write)
+    if sys.stdout:
+        stdout.stream.connect(sys.stdout.write)
+        stdout.flushed.connect(sys.stdout.flush)
+
+    stderr = TextStream()
+    error_writer = output_view.formated(color=Qt.red)
+    stderr.stream.connect(error_writer.write)
+    if sys.stderr:
+        stderr.stream.connect(sys.stderr.write)
+        stderr.flushed.connect(sys.stderr.flush)
+
+    sys.excepthook = ExceptHook(stream=stderr)
+
+    log.info("Entering main event loop.")
+    try:
+        with redirect_stderr(stderr), redirect_stdout(stdout):
```

Comment:
The commit message says tee instead of redirecting, while the function called here still says redirect.